### DISCUSSION TITLE
Lightning wallet

### DISCUSSION THREAD

**Discussion by EngEck** [raspiblitz/raspiblitz#4148] [Q&A]
How can I change or create a new wallet on the Raspiblitz?

**Comment by rootzoll** [comment]
There are multiple kind of wallets. A wallet in LND seems the most common, but there is is also Core Lightning and Bitcoin Core.

For LND - in the SSH menus go: `REPAIR > REPAIR-LND > RESET-LND` .. this will delete the old and create a new wallet with LND
